README.md

```diff
@@ -107,6 +106,24 @@ steps:
   run: ls -R
 ```
 
+# Download path output
+
+Not sure where your artifact was downloaded to? The `download-path` step output can help you out. Just make sure to be aware of the extra directory that is created if downloading all artifacts (no name specified).
```

Comment:
Minor, but I think we should make this description more informative (3rd person), it sort of sounds like an advertisement 😄

Reply:
Switched it up